admin/src/app/Components/hotel-cards/hotel-cards.component.html

```diff
@@ -0,0 +1,33 @@
+<div class="wrapper">
+    <app-hotels-filter></app-hotels-filter>
+    <div class="cards">
+        <mat-card class="card" *ngFor="let hotel of hotels">
+            <mat-grid-list cols="5" rowHeight="60px">
+                <mat-grid-tile [colspan]=3 [rowspan]=1>
```

Comment:
Always use `="value"`

```suggestion
                <mat-grid-tile [colspan]="3" [rowspan]="1">
```

```suggestion
                <mat-grid-tile colspan="3" rowspan="1">
```